README.md

```diff
@@ -111,6 +111,22 @@ appropriate to create a `PersistentVolumeClaim`/`PersistentVolume` and use it in
 Cassandra deployment does not support deleting pods or scaling down, as this might require
 administrative tasks that are dependent on the final deployment architecture.
 
+## Exposing Jaeger-Collector out of Cluster
+Collector is by default accessible only to services running inside the cluster. 
+The easiest is to expose HTTP `jaeger-collector-http` port via OpenShift Route:
```

Comment:
Maybe reword: "The easiest approach to expose the collector outside of the cluster is via the `jaeger-collector-http` HTTP port using an OpenShift Route:"